src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -990,16 +1009,36 @@ List<Exception> generalResolutionExceptions
                             // Log the reference which lost the conflict and the dependencies and source items which caused it.
                             LogReferenceDependenciesAndSourceItemsToStringBuilder(fusionName, conflictCandidate, logDependencies.AppendLine());
 
+                            string toOutput;
```

Comment:
```suggestion
                            string toOutput = StringBuilderCache.GetStringAndRelease(logConflict);
```

Both `if` and `else` do this as their first statement.